### ISSUE TITLE
Add option to save info about Panorama groups

### ISSUE THREAD

**Issue by kavalanche** [sienori/Tab-Session-Manager#217]
If groups are active,  then for each item save info about group it is placed in.

e.g.

"1": {
	"id": 1,
	"index": 1,
	"windowId": 3,
	...,
	"groupId": 1
},

**Comment by ZTHawk** [comment]
When I turn off the TSM setting "Restore last session" but enable the Firefox Core setting "Restore last session" then all Tabs and all groups are restored.
@sienori could you please try to restore all tab info?

**Comment by ZTHawk** [comment]
Pull request #313 could probably fix this issue.

**Comment by sienori** [comment]
Does "Panorama groups" mean the extension [Panorama View](https://addons.mozilla.org/en-US/firefox/addon/panorama-view/)?

The group data can only be accessed from the Panorama View.
Panorama View can access the group only from the same session as the session in which the group was saved.
In order to restore the same session, it is necessary to restore the session by using "Restore last session" of Firefox. It is impossible with Tab Session Manager.

**Comment by ZTHawk** [comment]
Yes, to be more precise the fork [Panorama Tab Groups](https://addons.mozilla.org/en-US/firefox/addon/panorama-tab-groups)
Restoring a session is what TSM is for. You are saying that TSM cannot restore sessions or only incomplete ones? Is there an API missing?

**Comment by lapineige** [comment]
In addition, is there any improvement that we could request to Panorama View / Panorama Tab Groups devs, that would make Tab Session Manager compatible with it ?
